src/common/models/customization/customization.ts

```diff
@@ -0,0 +1,10 @@
+export interface Customization {
+  headerBackground?: string;
+  customLogoSvg?: string;
+  externalView?: ExternalView;
+}
+
+export interface ExternalView {
+  title: string;
+  linkGenerator: string;
+}
```

Comment:
I think this should be a (immutable) class. There are a lot of checks to be done here.